### ISSUE TITLE
[Bug Report] Clipped Navigation Drawer is not below Toolbar

### ISSUE THREAD

**Issue by cawoodm** [vuetifyjs/vuetify#6547]
### Versions and Environment
**Vuetify:** 1.5.1
**Vue:** 2.6.6
**Browsers:** Chrome 72.0.3626.109
**OS:** Windows 10

### Steps to reproduce
According to the docs (https://vuetifyjs.com/en/components/navigation-drawers) a clipped drawer should reside under the toolbar. 

So, we make a navigation drawer with the clipped attribute and a toolbar below it.

Problem: the toolbar is on top of (overlayed over) the drawer.
https://codepen.io/anon/pen/BMbKpP

### Expected Behavior
Surely "under" means "down from" (2D) and is the desired behaviour instead of "below and hidden by"

[ toolbar            ]
[drawer]

### Actual Behavior
Drawer is obscured by toolbar.

### Reproduction Link
<a href="https://codepen.io/anon/pen/BMbKpP" target="_blank">https://codepen.io/anon/pen/BMbKpP</a>

### Other comments
Adding "clipped-left" (undocumented) pushes the drawer under the toolbar hiding part of the drawer:
https://codepen.io/anon/pen/xMBVzq

I'm sure there's a way to do it right but I've studied demos (e.g. https://vuetifyjs.com/en/examples/layouts/googleContacts) and can't see what's different so it's really confusing for beginners how this all behaves.

**Comment by KaelWD** [comment]
You need the `app` prop too. `clipped-left` tells the toolbar to ignore the drawer's width so there isn't an extra gap on the left.

**Comment by andrewjdavison** [comment]
adding the app property shifts the entire viewport to the right, although adding clipped and app DOES shift the elements down to the right level. 

I had assumed the cawoodm that adding clipped alone would shift the drawer below the app bar.

I douldn't find cource for the included google contact demo either, as the source link is returning a 404 from Github.

**Comment by MartinX3** [comment]
@KaelWD sorry for commenting in this old issue ticket, but where in the docs is this feature mentioned?
The only hint I found was here:
https://vuetifyjs.com/en/components/application/#application-components
`v-navigation-drawer: Can be placed on the left or right side of an application and can be configured to sit next to or below v-app-bar.`

But there is no example code or description how to use it.
Only with this issue ticket I found the `clipped` feature.

I also can't search for `clipped` in the docs, it always replaces it with `clicked`.
